htsget-config/README.md

```diff
@@ -275,6 +276,7 @@ regex = '.*'
 substitution_string = '$0'
 
 [resolvers.storage]
+type = 'S3'
```

Comment:
Hm, I'd probably name this `backend` instead? I think that type/objectype can lead to confusion even in this relatively narrow "storage" context?

Reply:
`backend` is a good name, more clear than `type`.